Security/Exception/LtiToolMessageExceptionHandlerInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace OAT\Bundle\Lti1p3Bundle\Security\Exception;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Throwable;
+
+interface LtiToolMessageExceptionHandlerInterface
+{
+    public function handle(Throwable $exception, Request $request): Response;
```

Comment:
In LTI 1.3 context, the Response should be always `RedirectResponse`, no?

Reply:
Youi can decide to interrupt the flow, and handle yourself error displaying.

So typhint with Response allow us regular and redirect responses, more flexible :)